nested-spire/nestedA/server/server.conf

```diff
@@ -0,0 +1,34 @@
+server {
+	bind_address = "0.0.0.0"
+	bind_port = "8081"
+	registration_uds_path = "/tmp/spire-registration.sock"
+	trust_domain = "example.org"
+	data_dir = "/opt/spire/data/server"
+	log_level = "DEBUG"
+	ca_ttl = "1h"
```

Comment:
can you change ca_ttl to something bigger? if we have the same ttl ca_ttl and svid_ttl will result in issues

Reply:
I missed that. Thanks for pointing it out.